__tests__/app.test.js

```diff
@@ -344,4 +344,15 @@ describe('app', () => {
         });
     });
   });
+
+  describe('/api/comments/:comment_id', () => {
```

Comment:
Lets also add our tests for 404 - non existant ID and 400 - invalid id

Reply:
Thanks Hannah, will do! :D